projects/ngneat/transloco-locale/README.md

```diff
@@ -0,0 +1,277 @@
+# Transloco Locale L10N
+
+This plugin provides localization(l10n) support for Transloco.
+
+> Localization refers to the adaptation of a product, application or document content to meet the language, cultural and other requirements of a specific target market (a locale).
+
+## Table of Contents
+
+- [Installation](#installation)
+- [Setup](#setup)
+- [Configuration](#configuration-options)
+- [Setting Locale](#setting-locale)
+  - [Locale Translation Files](#translation-files-name)
+  - [Language Locale Mapping](#language-locale-mapping)
+  - [Manually Setting Locale](#manually-setting-locale)
+- [Format Options](#locale-format-options)
+  - [Number Format Options](#number-format-options)
+  - [Date Format Options](#date-format-options)
+- [Service API](#service-api)
+- [Localization Pipes](#localization-pipes)
+  - [Date Pipe](#date-pipe)
+  - [Currency Pipe](#currency-pipe)
+  - [Decimal Pipe](#decimal-pipe)
+  - [Percent Pipe](#percent-pipe)
+  - [Browser Support](#browser-support)
+
+## Installation
+
+```
+npm i @ngneat/transloco-locale
+```
+
+### Setup
+
+Inject `TranslocoLocaleModule` along with `TranslocoModule` into `AppModule`:
+
+```typescript
+import { TranslocoLocaleModule } from '@ngneat/transloco-locale';
+
+@NgModule({
+  imports: [TranslocoModule, TranslocoLocaleModule.init()],
+  bootstrap: [AppComponent]
+})
+export class AppModule {}
+```
+
```

Comment:
How about an explanatory message here:

This library does not depend on locales provided by `@angular/common` package. It uses ECMAScript Internationalization API (**Intl**), which is available in all modern browsers and Internet Explorer 11. Please [check detailed browser support](#browser-support) for further details.